packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2044,7 +2044,7 @@ Fetches appLinks for all direct or indirect (via group membership) assigned appl
 
 | Parameter | Description  | Param Type | DataType | Required |
 | --------- | ------------ | ---------- | -------- | -------- |
-| id        | `id` of user | URL        | String   | TRUE     |
+| id        | `id`, `login`, or login shortname (as long as it is unambiguous) of user | URL        | String   | TRUE     |
```

Comment:
Should login shortname be in code font? `login shortname`? In earlier parts of the page https://developer.okta.com/docs/reference/api/users/#request-parameters-2 , it is in code format (might also need to update the other reference that is italicized.

Reply:
Fixed all occurrences.